SmartDeviceLink-iOS/SmartDeviceLink/SDLProtocol.m

```diff
@@ -28,6 +28,7 @@ @interface SDLProtocol () {
     dispatch_queue_t _sendQueue;
     SDLPrioritizedObjectCollection *_prioritizedCollection;
     NSMutableDictionary *_sessionIDs;
+    NSMutableDictionary *_hashIDs;
```

Comment:
Also, *technically* it should be `hashIds`, since "Id" is short for one word, not two. Not sure how we want to handle that since it's already done incorrectly with `sessionIDs`